pilot-app/src/components/form/wimForm.vue

```diff
@@ -0,0 +1,316 @@
+﻿<template>
+    <section>
+        <table class="formTable">
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
            <tbody>
```